cvmfs/publish/cmd_commit.cc

```diff
@@ -49,8 +49,7 @@ int CmdCommit::Main(const Options &options) {
     throw;
   }
 
-  if (!SwitchCredentials(settings->owner_uid(), settings->owner_gid(),
-                         false /* temporarily */))
+  if (!SwitchCredentials(settings->owner_uid(), settings->owner_gid(), false))
```

Comment:
I'd prefer to keep the inline documentation of the bool argument

Reply:
Done in #3730